roxie/ccd/ccddebug.cpp

```diff
@@ -114,6 +114,15 @@ class InputProbe : public CInterface, implements IFinalRoxieInput, implements IE
         else
             return in->queryConcreteInput(idx);
     }
+    virtual IEngineRowStream *queryConcreteOutputStream(unsigned idx)
+    {
+        return this;
+    }
+    virtual IStrandJunction *queryConcreteOutputJunction(unsigned idx) const
+    {
+        // MORE - not sure what is right here!
+        return in->queryConcreteOutputJunction(idx);
```

Comment:
I think this should currently return nullptr.  

The code above in getOutputStreams is assuming that there is a single input stream.  If that is broken (I suspect it will be fairly soon), then this class will need to save an array of streams, and have an array of output adaptors to rbe returned from this function.